docs/reference/sites-filter.md

```diff
@@ -0,0 +1,95 @@
+# Sites filter on platform landing page
+
+This document describes how the theme can be configured to display a filterable
+set of exhibits on the platform landing page.
+
+
+## Features
+
+This feature meets the following requirements:
+
+* One-click access to multiple sites around Omeka S, directly from the platform
+  homepage.
+* Provides the ability to filter sites based on arbitrary categories, without
+  involvement by theme developers.
+* Enables interaction via keyboard, mouse, and screen readers.
+
+
+## Dependencies
+
+The filterable interface relies upon vanilla javascript and the jQuery library,
+which is already loaded by the theme. The interface also requires the platform
+to host an item set containing records for each exhibit that will appear on the
+homepage. These records are defined below.
+
+### Exhibit records
+
+The first, and most complex, part of this feature is the maintenance of an item
```

Comment:
Can you add a sentence clarifying who is responsible for this item set maintenance? I assume it is not the theme developers but instead our primary stakeholders (aka Mikki's team?)

Reply:
Sure thing - and yes, those records are maintained by the platform staff (Mikki's team). DLS staff should know how it all functions in case something breaks, but Mikki's team are the ones to know what records are needed, when, etc.

Reply:
Yeah this was super helpful to understand how it all works (and why). And the new wording resolves my concern that it wasn't entirely clear who was responsible for the care and feeding of that list. Thanks!